src/index.ts

```diff
@@ -103,16 +106,18 @@ class FSMerger {
   _dirList: FSMerger.Node[];
   MAP: { [key: string]: FSMerger.FSMergerObject } | null;
   PREFIXINDEXMAP: { [key: number]: FSMerger.FSMergerObject };
+  LIST: FSMerger.FSMergerObject[];
   _atList: FSMerger[];
   fs: FSMerger.FS
 
   constructor(trees: FSMerger.Node[] | FSMerger.Node) {
     this._dirList = Array.isArray(trees) ? trees : [trees];
     this.MAP = null;
     this.PREFIXINDEXMAP = {};
+    this.LIST = [];
     this._atList = [];
     let self: FSMerger & {[key: string]: any} = this;
-    this.fs = new Proxy(nodefs, {
+    this.fs = <any>new Proxy(nodefs, {
```

Comment:
We have to cast through `any` here because this proxy doesn't have a single target object which confuses the typechecker.

Reply:
If we cast to `any`, do we get suggestions on available functions at the consumer of this `fs` proxy?

Reply:
@SparshithNR Yeah, the `fs` property is still strongly typed as before. Casting through `any` only disables the type-checking for this one assignment.